### PR TITLE
cooles improvement

### PR DESCRIPTION
**Pull request by indigo423**
Maximal improved

### CODE REVIEW

**Comment by Moorviper** [comment]
hab gelesen

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ the documentation for installing with/on ARCH-Linux will be available at: https:
 
 the documentation for installing on cubiuntu is available at: (in German)   
 http://www.vdr-portal.de/board18-vdr-hardware/board98-arm-co/123885-howto-working-allwinner-a20-cubieboard2-mit-vdr-softhd-device-und-streamdev/ 
+
+this is my cool readme feature
```
Comment (indigo423) [inline README.md:11]: Was soll das?

File: README.md [hunk 2]
```diff
@@ -7,3 +7,5 @@ the documentation for installing with/on ARCH-Linux will be available at: https:
 
 the documentation for installing on cubiuntu is available at: (in German)   
 http://www.vdr-portal.de/board18-vdr-hardware/board98-arm-co/123885-howto-working-allwinner-a20-cubieboard2-mit-vdr-softhd-device-und-streamdev/
```
Comment (indigo423) [inline README.md:9]: Muss das sein?

**Comment by indigo423** [comment]
Thanks for the example. :+1:

**Pull request closed without merging**